.github/CODEOWNERS

```diff
@@ -334,7 +334,7 @@
 /sdk/containerinstance/Microsoft.Azure.Management.ContainerInstance/ @dkkapur
 
 # ServiceLabel: %Container Registry
-/sdk/containerregistry/                                            @ShivangiReja @pallavit @annelo-msft
+/sdk/containerregistry/                                            @ShivangiReja @pallavit @annelo-msft @azsdk-acr
```

Comment:
This syntax isn't correct.   If this is a team, it needs the organization.   We probably want `@Azure/azsdk-acr`.   Same for the other teams being added.

Reply:
Thanks! My bad - updated.